### PR TITLE
Removed IMU Velocity Computations

### PR DESCRIPTION
**Pull request by lymacasm**
IMU velocity has horrible drift, which is nearly impossible to deal with in the current system. This goes so far as to say that the control of the sub would be improved when the control system for linear velocities is removed. Velocity Z will be controlled using depth from the depth sensor. IMU now doesn't publish velocity.

Added a topic to publish depth.

### CODE REVIEW

File: ros/src/navigation/src/control_system.cpp [hunk 1]
```diff
@@ -203,19 +176,20 @@ bool control_system::control_enable_service(ControlEnReq &req, ControlEnRes &res
 
 void control_system::compute_output_vectors(navigation::nav &msg)
 {       
-    if(depth_calibrated)
+    //if(depth_calibrated)
+    if(true)
```
Comment (asinha94) [inline ros/src/navigation/src/control_system.cpp]: remove this please. I hate unnecessary indentation

File: ros/src/peripherals/src/imu.cpp [hunk 2]
```diff
@@ -411,61 +409,36 @@ int main(int argc, char ** argv)
     ros::Publisher pub = nh.advertise<peripherals::imu>(topic_name, topic_buffer_size);
 
     // Update velocity "updates_per_publish" times for every topic publish
-    ros::Rate r(publish_rate * updates_per_publish);
+    ros::Rate r(publish_rate);
     uint16_t count = 0;
```
Comment (asinha94) [inline ros/src/peripherals/src/imu.cpp]: do you still need count?

**Review by asinha94 — CHANGES_REQUESTED** [review]

File: ros/src/navigation/src/control_system.cpp (hunk 1 above)
Comment (lymacasm) [reply ros/src/navigation/src/control_system.cpp]: Sorry, I just had the if(true) for testing, I'll change it back to the way it was.

File: ros/src/peripherals/src/imu.cpp (hunk 2 above)
Comment (lymacasm) [reply ros/src/peripherals/src/imu.cpp]: I do not!

File: ros/src/navigation/src/control_system.cpp [hunk 3]
```diff
@@ -272,6 +255,16 @@ int main(int argc, char ** argv)
         // Publish the vectors
         pub_vectors.publish(output_vectors);
 
+        if((++count %= loops_per_param_update) == 0)
```
Comment (asinha94) [inline ros/src/navigation/src/control_system.cpp]: change to count == 10 then set count = 0;

**Review by asinha94 — CHANGES_REQUESTED** [review]

File: ros/src/navigation/src/control_system.cpp (hunk 3 above)
Comment (asinha94) [reply ros/src/navigation/src/control_system.cpp]: put the count at the top of the if statement too...if you're worried about efficiency

**Review by asinha94 — APPROVED** [review]

**Pull request merged**